previews/anchor/forms_preview.rb

```diff
@@ -12,5 +12,14 @@ def with_errors
 
       render_with_template(template: "forms/default", locals: { model: })
     end
+
+    def select_with_descriptions
```

Comment:
This made me think this was a `<select>` element.
```suggestion
    def radio_button_collection
```

Reply:
🤦 🤣 I have no idea what what happened there.